spec/components/app_session_needs_review_warning_component_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+describe AppSessionNeedsReviewWarningComponent do
+  subject(:rendered) { render_inline(component) }
+
+  let(:component) { described_class.new(session:) }
+  let(:session) { create(:session) }
+
+  describe "#render?" do
```

Comment:
Inside here we can define this as the subject of the tests:

```rb 
subject { component.render? }
```

And then shorten the amount code needed below by using the short-hand syntax:

```rb
it { should be(true) }
```